fe/fe-core/src/main/java/com/starrocks/sql/parser/StarRocks.g4

```diff
@@ -546,6 +547,10 @@ dropTableStatement
     : DROP TEMPORARY? TABLE (IF EXISTS)? qualifiedName FORCE?
     ;
 
+cleanTemporaryTableStatement
+    : CLEAN TEMPORARY TABLE ON SESSION string
```

Comment:
> Since we have background cleaner, when should we use this sql?

The admin can use it to forcefully delete the temporary table created by a session. For example, when rolling back to an old version, because the metadata is not forward compatible, we need to ensure that there are no temporary tables in the metadata image. At this time, we can use this command to clean up and then regenerate the Image.